### PR TITLE
add custom css file for sofia

### PR DESCRIPTION
**Pull request by sdubinsky**

### CODE REVIEW

**Review by nvella — APPROVED** [review]
LGTM 👍 Mastodon does have a theming system as demonstrated [in this repo](https://github.com/trwnh/mastomods), however I'm not sure how involved it gets to set it up.

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -35,6 +35,7 @@
 
     = stylesheet_link_tag '/inert.css', skip_pipeline: true, media: 'all', id: 'inert-style'
     = stylesheet_link_tag custom_css_path, skip_pipeline: true, host: root_url, media: 'all'
+    = stylesheet_link_tag '/custom.css', skip_pipeline: true, media: 'all', id: 'custom-style'
```
Comment (sofiarivas) [inline app/views/layouts/application.html.haml:38]: Just to confirm, this means we are importing the custom css file and we no longer need to add an import statement over at:

 > _Appearance_ > _Custom CSS_ input? `(@import url("./custom.css");`
 
 right?

**Review by sofiarivas — APPROVED** [review]

File: app/views/layouts/application.html.haml (hunk 1 above)
Comment (sdubinsky) [reply app/views/layouts/application.html.haml:38]: Correct.

**Comment by sdubinsky** [comment]
@nvella that basically replaces the entire css system with the custom theme.   This is a less-intrusive way of doing it.

**Pull request merged**